### DISCUSSION TITLE
Amazon Carrier Central API documentation

### DISCUSSION THREAD

**Discussion by advancebyanymeans** [amzn/selling-partner-api-models#3614] [General]
Hi, I am new to this forum. Is there any resource like 'Amazon Carrier Central API documentation' that has APIs to get information on Amazon carrier central? Any help will be greatly appreciated.

**Comment by threeaccents** [comment]
@marslang1994 Did you ever find out if they offer an API?

**Comment by mott7** [comment]
Any news on this?

**Reply by threeaccents** [reply]
I have nothing.
